types/jss/jss-tests.ts

```diff
@@ -7,13 +7,25 @@ import {
 
 const jss = createJSS().setup({});
 
-const styleSheet = jss.createStyleSheet(
+const styleSheet = jss.createStyleSheet<string>(
 	{
 		ruleWithMockObservable: {
-			subscribe() {}
+			subscribe: observer => {
+				const next = typeof observer === 'function' ? observer : observer.next;
+				next({ background: 'blue', display: 'flex' });
+				next({ invalidKey: 'blueish' }); // $ExpectError
+
+				// only kebab case allowed in observables
+				next({ 'align-items': 'center' });
+				next({ alignItems: 'center' }); // $ExpectError
+				return {
+					unsubscribe() {}
+				};
+			}
```

Comment:
Do `alignItems` and `invalidKey` actually error?

I had mentioned in [this comment](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/23116/files/d0adcf2e8f3235e98ca35ef9481d349ac82fad59#r163411489) that I thought passing an observable of the wrong shape to `createStyleSheet` might be a better test, but I never saw any follow up on it.

Reply:
>Do `alignItems` and `invalidKey` actually error?

Yes, the build would be failing otherwise.